src/views/TechnicalAssistance/AdminHome/index.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { Link, Route, useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import { Grid, GridContainer, IconArrowBack } from '@trussworks/react-uswds';
+import classNames from 'classnames';
+
+import PageLoading from 'components/PageLoading';
+import GetTrbRequestQuery from 'queries/GetTrbRequestQuery';
+import {
+  GetTrbRequest,
+  GetTrbRequestVariables
+} from 'queries/types/GetTrbRequest';
+import { AppState } from 'reducers/rootReducer';
+import user from 'utils/user';
+import AccordionNavigation from 'views/GovernanceReviewTeam/AccordionNavigation';
+import NotFound from 'views/NotFound';
+
+import subNavItems from './subNavItems';
+import Summary from './Summary';
+
+import './index.scss';
+
+const SideNavigation = ({
+  activePage,
+  trbRequestId
+}: {
+  activePage: string;
+  trbRequestId: string;
+}) => {
+  const { t } = useTranslation('technicalAssistance');
+  return (
+    <nav>
+      <ul className="trb-admin__nav-list usa-list usa-list--unstyled">
+        <li className="margin-bottom-4">
+          <Link to="/" className="display-flex flex-align-center">
+            <IconArrowBack className="margin-right-1" aria-hidden />
+            {t('adminHome.subnav.back')}
+          </Link>
+        </li>
+        {subNavItems(trbRequestId).map(({ route, text, groupEnd }) => {
+          const isActivePage: boolean = route.split('/')[3] === activePage;
+          return (
+            <li
+              key={text}
+              className={classNames(
+                'padding-y-1',
+                'trb-admin__nav-link',
+                'hover:bg-base-lightest',
+                {
+                  'trb-admin__nav-link--active': isActivePage,
+                  'border-bottom-1px border-disabled-light': groupEnd
+                }
+              )}
+            >
+              <Link
+                to={route}
+                className={classNames('text-no-underline', 'padding-y-05', {
+                  'text-base-darkest': !isActivePage,
+                  'text-primary': isActivePage
+                })}
+              >
+                {t(text)}
+              </Link>
+            </li>
+          );
+        })}
+      </ul>
+    </nav>
+  );
+};
+
+export default function AdminHome() {
+  /** Current user groups */
+  const userGroups = useSelector((state: AppState) => state.auth.groups);
+  /** Whether or not current user is set */
+  const isUserSet = useSelector((state: AppState) => state.auth.isUserSet);
+
```

Comment:
Just a stylistic note, this could be condensed to something like:

```
const { groups: userGroups, isUserSet } = useSelector(
    (state: AppState) => state.auth
);
```